### DISCUSSION TITLE
Jammy chiseled extra for .NET 6

### DISCUSSION THREAD

**Discussion by u001617** [dotnet/dotnet-docker#5077] [General]
Hello there!

I am doing a POC using jammy chiseled extra, and I noticed that the .NET 6 version is only available as preview (nightly).

`mcr.microsoft.com/dotnet/aspnet:6.0.25-jammy-chiseled-extra` doesn't exist, but `mcr.microsoft.com/dotnet/nightly/aspnet:6.0.25-jammy-chiseled-extra` does.

Is there a reason not to use it yet in production environment? If not, is there a estimated date for releasing it?

Thanks in advance :)

**Comment by lbussell** [comment]
Hello, the nightly images are not for use in production as we don't make any guarantees about the security content of those images. If your app is just a proof-of-concept and/or not security critical, we always appreciate users trying out new features in the nightly repo and providing feedback!

We plan to release the ASP.NET Core and .NET Runtime Ubuntu Chiseled `extra` images for .NET 6 and 7 with the next .NET servicing release, which should be 2024-01-09 (next Tuesday). This was mentioned in https://github.com/dotnet/dotnet-docker/issues/5014#issuecomment-1834393550.

**Comment by u001617** [comment]
I saw that comment but was unsure if it was about .NET 6. Now it's clear!

Thank you for pointing it out!

**Comment by lbussell** [comment]
@u001617, the Chiseled "extra" ASP.NET and Runtime images are now available - the tag you are probably looking for is `mcr.microsoft.com/dotnet/aspnet:6.0.26-jammy-chiseled-extra`
